msdbook/tests/test_utils.py

```diff
@@ -23,15 +23,14 @@ def sample_data():
     })
 
     return df
+
+# Define column names as variables for easy reference
 predictor1 = 'Predictor1'
 predictor2 = 'Predictor2'
 interaction = 'Interaction'
 intercept = 'Intercept'
 success = 'Success'
```

Comment:
You should define these vars above `sample_data()` and use the vars in the function